### ISSUE TITLE
Don't allow email actions for accounts without their own email account

### ISSUE THREAD

**Issue by nicpottier** [rapidpro/rapidpro#1152]
A useful action in flows is the "email" action. This lets someone generate a new email from flow variables and send it along. Definitely one of the more used actions.

Problem is that users can abuse this or misconfigure it in ways that have repercussions on the hoster. Specifically AWS will shut down email across your entire region if you start bouncing at a high enough rate, something which someone could cause by typo-ing an email address in their account and sending a large volume of emails.

Even in the best case scenario of everything being kosher with email addresses, users could use a flow to send spam or the like from the hoster's domain, completely wrecking their email reputation and causing larger issues.

Long story short, if people want to send emails, they should do it from their own accounts that they configure. We already have the ability to do this from the account page, we just need to point people there when they try to create a send-email action.

**Comment by rowanseymour** [comment]
Should we get started on this by disallowing this for new orgs, and warning existing orgs that they need to start finding an email service? Is worrying that we might still get ourselves into a state where all email is blocked because of one org.

**Comment by nicpottier** [comment]
This is still an issue and we're getting in some hot water due to people using invalid email addresses.

It is a bit painful for the average user to plug in their own email service, even gmail is painful due to the need of creating a "application password" for use with RapidPro.

Another way around bounces would be to validate ownership of the email address when using the "default" address. IE, the send email action would only allow you to send to an email address you verified using a one time code we send you. If people want to send to arbitrary email addresses instead then they would instead need to hook in their own SMTP server.
